### ISSUE TITLE
584107FE - Assault Heroes

### ISSUE THREAD

**Issue by Blackbird88** [xenia-project/game-compatibility#125]
[Marketplace](http://marketplace.xbox.com/en-US/Product/Assault-Heroes/66acd000-77fe-1000-9115-d802584107fe)

* Gameplay with broken graphics and half-menu - Release
* Crashes after loading bar is full on new game due to assert erros - Debug

[Log](https://gist.github.com/Blackbird88/bf4360fead295644d06b)

![image](https://cloud.githubusercontent.com/assets/4414625/9107388/306573b6-3c27-11e5-98d2-3d8ccc452914.png)

**state-gameplay** and **gpu-corrupt-drawing**

**Comment by Razzile** [comment]
I'm unsure what the difference between state-gameplay and state-playable are. I think gameplay is for if it reaches in game but you can't actually play it or something.

**Comment by Blackbird88** [comment]
I thought it's same as PCSX2/Dolphin In-game which is playable with major glitches, but dunno since there is no description anywhere.

**Comment by Sinoxan** [comment]
Indeed, state-gameplay if it gets into the main gameplay but with half the screen missing I wouldn't class this as playable.

**Comment by bdeimos** [comment]
Tested on (https://github.com/xenia-project/xenia/commit/f89b4626a37dd4d1e5a817909252d3d4dd054299)

Menus - OK. Gameplay is black screen.

(gpu-missing-drawing)

[xenia.zip](https://github.com/xenia-project/game-compatibility/files/2148540/xenia.zip)
